external/quarkus_openshift/README.md

```diff
@@ -0,0 +1,31 @@
+# External quarkus_openshift Tests
+
+Third Party container tests help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue [#172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests. Quarkus Openshift tests are pulled from the [quarkus-openshift-test-suite](https://github.com/quarkus-qe/quarkus-openshift-test-suite.git) repository. These tests require an Openshift cluster environment to run.
```

Comment:
Please change the first part of the sentence to be:
Quarkus_openshift tests are part of the external third-party application tests that help verify that the Adoptium binaries are good...